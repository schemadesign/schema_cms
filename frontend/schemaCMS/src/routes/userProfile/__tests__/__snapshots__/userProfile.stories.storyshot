// Jest Snapshot v1, https://goo.gl/fbAQLP

exports[`Storyshots UserProfile Default 1`] = `
<div
  className="sc-feJyhm fRrBax"
>
  <div
    className="sc-htpNat KtFsv"
  >
    <div
      style={
        Object {
          "alignItems": "flex-start",
          "color": "#1d1d20",
          "display": "flex",
          "flexDirection": "row",
          "justifyContent": "space-between",
          "padding": "18px 0",
          "width": "100%",
        }
      }
    >
      <div>
        <div
          className="sc-bxivhb bUZGRr"
        >
          <h2
            id="projectsPageTitle"
            style={
              Object {
                "fontFamily": "Inter, \\"Helvetica Neue\\", Helvetica, Arial, sans-serif",
                "fontSize": "24px",
                "fontWeight": 100,
                "margin": 0,
              }
            }
          >
            Admin Settings
          </h2>
          <h1
            id="projectsPageSubtitle"
            style={
              Object {
                "fontFamily": "Inter, \\"Helvetica Neue\\", Helvetica, Arial, sans-serif",
                "fontSize": "24px",
                "fontWeight": 600,
                "letterSpacing": "-0.12px",
                "lineHight": 1.25,
                "margin": 0,
              }
            }
          >
            Profile
          </h1>
        </div>
      </div>
      <button
        onClick={[Function]}
        style={
          Object {
            "backgroundColor": "transparent",
            "border": "none",
            "borderRadius": "48px",
            "color": "#1d1d20",
            "cursor": "pointer",
            "display": "inline-block",
            "fontSize": "18px",
            "margin": "0",
            "minHeight": "48px",
            "outline": "none",
            "padding": "0",
            "position": "absolute",
            "right": 0,
            "top": 6,
          }
        }
      >
        <svg
          style={
            Object {
              "fill": "#1d1d20",
              "height": "60px",
              "width": "60px",
            }
          }
          viewBox="0 0 60 60"
        >
          <path
            d="M30 37a3 3 0 110 6 3 3 0 010-6zm0-10a3 3 0 110 6 3 3 0 010-6zm0-10a3 3 0 110 6 3 3 0 010-6z"
            fillRule="evenodd"
          />
        </svg>
      </button>
    </div>
    <div
      style={
        Object {
          "backgroundColor": "#fff",
          "color": "#1d1d20",
          "maxWidth": null,
          "minHeight": "100vh",
          "padding": "18px 20px 27px",
          "position": "fixed",
          "right": 0,
          "textAlign": "left",
          "top": 0,
          "transform": "translateX(100%)",
          "transition": "transform 400ms cubic-bezier(0.86, 0, 0.07, 1), visibility 0s linear 400ms",
          "visibility": 0,
          "width": "100%",
          "zIndex": 9999,
        }
      }
    >
      <div
        className="sc-ifAKCX cFlEyZ"
      >
        <div
          className="sc-bxivhb bUZGRr"
        >
          <h2
            id="projectsPageTitle"
            style={
              Object {
                "fontFamily": "Inter, \\"Helvetica Neue\\", Helvetica, Arial, sans-serif",
                "fontSize": "24px",
                "fontWeight": 100,
                "margin": 0,
              }
            }
          >
            Admin Settings
          </h2>
          <h1
            id="projectsPageSubtitle"
            style={
              Object {
                "fontFamily": "Inter, \\"Helvetica Neue\\", Helvetica, Arial, sans-serif",
                "fontSize": "24px",
                "fontWeight": 600,
                "letterSpacing": "-0.12px",
                "lineHight": 1.25,
                "margin": 0,
              }
            }
          >
            Profile
          </h1>
        </div>
      </div>
      <div
        className="sc-EHOje qQtBf"
      >
        <ul
          className="sc-bZQynM sc-gzVnrw JqJZP"
        />
        <ul
          className="sc-bZQynM sc-htoDjs LPVfx"
        />
      </div>
      <button
        onClick={[Function]}
        style={
          Object {
            "backgroundColor": "#fff",
            "border": "none",
            "borderRadius": "48px",
            "color": "#1d1d20",
            "cursor": "pointer",
            "display": "inline-block",
            "fontSize": "18px",
            "margin": "0",
            "minHeight": "48px",
            "outline": "none",
            "padding": "0",
            "position": "absolute",
            "right": 0,
            "top": "6px",
          }
        }
      >
        <svg
          style={
            Object {
              "fill": "#1d1d20",
              "height": "60px",
              "width": "60px",
            }
          }
          viewBox="0 0 60 60"
        >
          <path
            d="M39.192 17.98l2.829 2.828L32.828 30l9.193 9.192-2.829 2.829L30 32.828l-9.192 9.193-2.829-2.829L27.172 30l-9.193-9.192 2.829-2.829L30 27.172l9.192-9.193z"
            fillRule="evenodd"
          />
        </svg>
      </button>
    </div>
  </div>
  <form
    className="sc-iELTvK fyBIax"
    onSubmit={[Function]}
  >
    <div
      className="sc-gqjmRU hnZXkc"
    >
      <div
        style={
          Object {
            "display": "flex",
            "flexDirection": "column",
            "maxWidth": "320px",
            "paddingBottom": "30px",
            "position": "relative",
          }
        }
      >
        <label
          htmlFor="firstName"
          style={
            Object {
              "border": "none",
              "borderTop": "1px solid #F0EFEF",
              "color": "#a6a7aa",
              "display": "block",
              "fontSize": "14px",
              "outline": "none",
              "paddingTop": "8px",
              "width": "100%",
            }
          }
        >
          First Name
        </label>
        <input
          id="firstName"
          name="firstName"
          onChange={[Function]}
          style={
            Object {
              "backgroundColor": "transparent",
              "border": "none",
              "color": "#1d1d20",
              "fontSize": "18px",
              "lineHeight": "24px",
              "outline": "none",
            }
          }
          type="text"
          value=""
        />
        <div
          style={
            Object {
              "position": "absolute",
              "right": 0,
              "top": 0,
            }
          }
        />
      </div>
    </div>
    <div
      className="sc-gqjmRU hnZXkc"
    >
      <div
        style={
          Object {
            "display": "flex",
            "flexDirection": "column",
            "maxWidth": "320px",
            "paddingBottom": "30px",
            "position": "relative",
          }
        }
      >
        <label
          htmlFor="lastName"
          style={
            Object {
              "border": "none",
              "borderTop": "1px solid #F0EFEF",
              "color": "#a6a7aa",
              "display": "block",
              "fontSize": "14px",
              "outline": "none",
              "paddingTop": "8px",
              "width": "100%",
            }
          }
        >
          Last Name
        </label>
        <input
          id="lastName"
          name="lastName"
          onChange={[Function]}
          style={
            Object {
              "backgroundColor": "transparent",
              "border": "none",
              "color": "#1d1d20",
              "fontSize": "18px",
              "lineHeight": "24px",
              "outline": "none",
            }
          }
          type="text"
          value=""
        />
        <div
          style={
            Object {
              "position": "absolute",
              "right": 0,
              "top": 0,
            }
          }
        />
      </div>
    </div>
    <div
      className="sc-gqjmRU hnZXkc"
    >
      <div
        style={
          Object {
            "display": "flex",
            "flexDirection": "column",
            "maxWidth": "320px",
            "paddingBottom": "30px",
            "position": "relative",
          }
        }
      >
        <label
          htmlFor="email"
          style={
            Object {
              "border": "none",
              "borderTop": "1px solid #F0EFEF",
              "color": "#a6a7aa",
              "display": "block",
              "fontSize": "14px",
              "outline": "none",
              "paddingTop": "8px",
              "width": "100%",
            }
          }
        >
          Email
        </label>
        <input
          disabled={true}
          id="email"
          name="email"
          onChange={[Function]}
          style={
            Object {
              "backgroundColor": "transparent",
              "border": "none",
              "color": "#1d1d20",
              "fontSize": "18px",
              "lineHeight": "24px",
              "outline": "none",
            }
          }
          type="text"
          value=""
        />
        <div
          style={
            Object {
              "position": "absolute",
              "right": 0,
              "top": 0,
            }
          }
        />
      </div>
    </div>
    <button
      style={
        Object {
          "backgroundColor": "#f0efef",
          "border": "none",
          "borderRadius": "48px",
          "color": "#1d1d20",
          "cursor": "pointer",
          "display": "inline-block",
          "fontSize": "18px",
          "margin": "0",
          "marginRight": "20px",
          "minHeight": "48px",
          "outline": "none",
          "padding": "0 30px",
        }
      }
    >
      Save
    </button>
  </form>
  <div
    className="sc-jwKygS jgPAfV"
  >
    <a
      className="sc-cmTdod clnUB"
      href="/reset-password"
      onClick={[Function]}
    >
      Reset Password
    </a>
    <a
      className="sc-cmTdod clnUB"
      href="/logout"
      onClick={[Function]}
    >
      Log out
    </a>
  </div>
</div>
`;
