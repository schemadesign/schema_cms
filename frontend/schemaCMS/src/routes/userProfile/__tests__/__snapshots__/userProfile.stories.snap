// Jest Snapshot v1, https://goo.gl/fbAQLP

exports[`Storyshots UserProfile Default 1`] = `
<div
  className="sc-brqgnP SqPLV"
>
  <div
    style={
      Object {
        "alignItems": "flex-start",
        "color": "#1d1d20",
        "display": "flex",
        "flexDirection": "row",
        "justifyContent": "space-between",
        "padding": "18px 0",
        "width": "100%",
      }
    }
  >
    <div>
      <h2
        style={
          Object {
            "fontFamily": "Inter, \\"Helvetica Neue\\", Helvetica, Arial, sans-serif",
            "fontSize": "24px",
            "fontWeight": 100,
            "margin": 0,
          }
        }
      >
        Admin Settings
      </h2>
      <h1
        style={
          Object {
            "fontFamily": "Inter, \\"Helvetica Neue\\", Helvetica, Arial, sans-serif",
            "fontSize": "24px",
            "fontWeight": 600,
            "letterSpacing": "-0.12px",
            "lineHight": 1.25,
            "margin": 0,
          }
        }
      >
        Profile
      </h1>
    </div>
    <button
      onClick={[Function]}
      style={
        Object {
          "backgroundColor": "transparent",
          "border": "none",
          "borderRadius": "48px",
          "color": "#1d1d20",
          "cursor": "pointer",
          "display": "inline-block",
          "fontSize": "18px",
          "margin": "0",
          "minHeight": "48px",
          "outline": "none",
          "padding": "0",
          "position": "absolute",
          "right": 0,
          "top": 6,
        }
      }
    >
      <svg
        style={
          Object {
            "fill": "#1d1d20",
            "height": "60px",
            "width": "60px",
          }
        }
        viewBox="0 0 60 60"
      >
        <path
          d="M30 37a3 3 0 110 6 3 3 0 010-6zm0-10a3 3 0 110 6 3 3 0 010-6zm0-10a3 3 0 110 6 3 3 0 010-6z"
          fillRule="evenodd"
        />
      </svg>
    </button>
  </div>
  <form
    className="sc-cMljjf eTbdzc"
    onSubmit={[Function]}
  >
    <div
      className="sc-bwzfXH uTxCW"
    >
      <div
        style={
          Object {
            "display": "flex",
            "flexDirection": "column",
            "maxWidth": "320px",
            "paddingBottom": "30px",
            "position": "relative",
          }
        }
      >
        <input
          handleChange={[Function]}
          handleSubmit={[Function]}
          id="firstName"
          intl={
            Object {
              "formatMessage": [Function],
            }
          }
          name="firstName"
          onChange={[Function]}
          style={
            Object {
              "backgroundColor": "transparent",
              "border": "none",
              "color": "#1d1d20",
              "fontSize": "18px",
              "lineHeight": "24px",
              "outline": "none",
            }
          }
          type="text"
          value=""
          values={
            Object {
              "email": "",
              "firstName": "",
              "lastName": "",
            }
          }
        />
        <div
          style={
            Object {
              "position": "absolute",
              "right": 0,
              "top": 0,
            }
          }
        />
      </div>
    </div>
    <div
      className="sc-bwzfXH uTxCW"
    >
      <div
        style={
          Object {
            "display": "flex",
            "flexDirection": "column",
            "maxWidth": "320px",
            "paddingBottom": "30px",
            "position": "relative",
          }
        }
      >
        <input
          handleChange={[Function]}
          handleSubmit={[Function]}
          id="lastName"
          intl={
            Object {
              "formatMessage": [Function],
            }
          }
          name="lastName"
          onChange={[Function]}
          style={
            Object {
              "backgroundColor": "transparent",
              "border": "none",
              "color": "#1d1d20",
              "fontSize": "18px",
              "lineHeight": "24px",
              "outline": "none",
            }
          }
          type="text"
          value=""
          values={
            Object {
              "email": "",
              "firstName": "",
              "lastName": "",
            }
          }
        />
        <div
          style={
            Object {
              "position": "absolute",
              "right": 0,
              "top": 0,
            }
          }
        />
      </div>
    </div>
    <div
      className="sc-bwzfXH uTxCW"
    >
      <div
        style={
          Object {
            "display": "flex",
            "flexDirection": "column",
            "maxWidth": "320px",
            "paddingBottom": "30px",
            "position": "relative",
          }
        }
      >
        <input
          handleChange={[Function]}
          handleSubmit={[Function]}
          id="email"
          intl={
            Object {
              "formatMessage": [Function],
            }
          }
          name="email"
          onChange={[Function]}
          style={
            Object {
              "backgroundColor": "transparent",
              "border": "none",
              "color": "#1d1d20",
              "fontSize": "18px",
              "lineHeight": "24px",
              "outline": "none",
            }
          }
          type="text"
          value=""
          values={
            Object {
              "email": "",
              "firstName": "",
              "lastName": "",
            }
          }
        />
        <div
          style={
            Object {
              "position": "absolute",
              "right": 0,
              "top": 0,
            }
          }
        />
      </div>
    </div>
  </form>
</div>
`;
